blocks/multiplatform/cta-block/cta-block.tsx

```diff
@@ -0,0 +1,27 @@
+import { useDL } from '@jetbrains/kotlin-web-site-ui/out/components/breakpoints-v2';
+import { Button } from '@rescui/button';
+import { useTextStyles } from '@rescui/typography';
+import cn from 'classnames';
+import Img from 'next/image';
+import ctaGraphics from './images/cta-graphics.webp';
+import styles from './cta-block.module.css';
+
+export const CtaBlock = ({ url }: { url: string }) => {
+    const isDL = useDL();
+    const textCn = useTextStyles();
+    return (
+        <div className={styles.wrapper}>
+            <div className="ktl-layout ktl-layout--center">
+                <div className={styles.content}>
+                <h2 className={cn(textCn('rs-h1'), styles.title)}>
```

Comment:
That's wrong size on `TS`. `rs-h1` transform on `ml` but in figma it should be on `ts`

<img width="1139" height="237" alt="Image" src="https://github.com/user-attachments/assets/081e3ff6-b375-4a8b-ae13-8746d3d839ef" />